lib/cli.js

```diff
@@ -41,6 +41,16 @@ var processArgs = function (argv, options, fs, path) {
     options.failOnEmptyTestSuite = options.failOnEmptyTestSuite === 'true'
   }
 
+  if (helper.isString(options.formatError)) {
```

Comment:
This whole validation block, should be part of [`lib/config.js`](https://github.com/karma-runner/karma/blob/master/lib/config.js) as otherwise this option is not fully usable from the api exposed to thing like grunt and gulp.